pkg/kubernetes/scripts/kube-client.js

```diff
@@ -64,6 +64,7 @@
     ]);
 
     var NAME_RE = /^[a-z0-9]([-a-z0-9]*[a-z0-9])?$/;
```

Comment:
Can you add that reasoning as a comment? Maybe even link to the user name format documentation. In a week or two I'll probably think `NAME_RE` and `USER_NAME_RE` are confusing...